ui/src/app/components/object-detail/object-detail.component.ts

```diff
@@ -747,6 +799,159 @@ export class ObjectDetailComponent implements OnInit {
     this.addedPkColumnName = columnName
   }
 
+  addCcColumn() {
+    const index = this.ccData.length;
+  
+    this.ccData.push({
+      spSno: (index + 1).toString(),
+      spConstraintName: `Constraint_name${index + 1}`,
+      spCondition: '',
+      srcSno: '',
+      srcCondition: '',
+      srcConstraintName: '',
+      deleteIndex: `cc${index + 1}`,
+    });
+  
+    this.setCCRows();
+  }
+    
+  dropCc(element: any) {
+    const index = this.ccData.findIndex(item => item.deleteIndex === element.value.deleteIndex);
```

Comment:
Similar to other items we should open a dialogue box

Reply:
for drop in other section also there is no dialogue box is shown have check in COLUMNS tab